core/sql/bin/SqlciErrors.txt

```diff
@@ -515,6 +515,7 @@
 1718 ZZZZZ 99999 BEGINNER MINOR DBADMIN --- unused ---
 1719 ZZZZZ 99999 BEGINNER MINOR DBADMIN Access Type '$0~string0' is not supported.
 1720 ZZZZZ 99999 BEGINNER MINOR DBADMIN Isolation Level '$0~string0' is not supported.
+1722 ZZZZZ 99999 BEGINNER MINOR DBADMIN cannot set comment on native HBASE table.
```

Comment:
First word should  be capitalized. Also, HBASE should be HBase.

Also, please add this message to the Trafodion Messages Guide. The source can be found in the trafodion/docs/messages_guide directory.